src/main/java/org/testeditor/web/xtext/index/resources/IGlobalScopeResource.xtend

```diff
@@ -9,6 +9,37 @@ import org.eclipse.xtext.resource.IEObjectDescription
 
 interface IGlobalScopeResource {
```

Comment:
No "I" before Interfaces. This is obsolet. Also Eclipse marks this as deprecated.